src/components/quote/README.md

```diff
@@ -0,0 +1,47 @@
+# Quote
+Call prominent attention to a quotation using large text and a citation
+
+## When to use
+Use the quote component when you have a quotation that you would like to display prominently in your page. This could include:
+
+- A sentence or short phrase quoted by an important person
```

Comment:
I feel 'important' is a somewhat debatable term. Consider rephrasing to something like, 'A sentence or short phrase, attributed to a specific person'.